### PR TITLE
Add support for elerea-2.9.0

### PR DESCRIPTION
**Pull request by maoe**
elerea-2.9.0 renamed `until` to `till`. This PR adds a compatibility layer to support `till` even with older elerea.

### CODE REVIEW

**Comment by fumieval** [comment]
LGTM

File: FRP/Elerea/Simple/Compat.hs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{-# LANGUAGE CPP #-}
+module FRP.Elerea.Simple.Compat
+    ( module FRP.Elerea.Simple
+    , module FRP.Elerea.Simple.Compat
+    ) where
+import FRP.Elerea.Simple
+
+#if !MIN_VERSION_elerea(2, 9, 0)
+till :: Signal Bool -> SignalGen (Signal Bool)
+till = unless
```
Comment (asayers) [inline FRP/Elerea/Simple/Compat.hs]: Should this be `until`?

Comment (maoe) [inline FRP/Elerea/Simple/Compat.hs]: Oops. Good catch. I should fix the Travis CI to run tests on multiple versions of GHC.

**Comment by maoe** [comment]
I've updated `.travis.yml` so we can run the tests with a couple of versions of elerea and GHC.

I put GHC 8.0.1 to `allow_failures` because `cabal` [fails to resolve dependencies](https://travis-ci.org/tsurucapital/euphoria/jobs/148184640#L240-L259) for `enummapset-th`. There's a [PR](https://github.com/tsurucapital/enummapset-th/pull/5) to fix the failure.

**Comment by maoe** [comment]
A new version of `enummapset-th` has been released. I dropped `allow_failures` for GHC 8.0.1.

**Pull request merged**